contracts/pool/AbstractPool.sol

```diff
@@ -261,6 +261,20 @@ abstract contract AbstractPool is PoolFDT, Pausable, Ownable {
         );
     }
 
+    function setSecondaryLiquidityAsset(address _liquidityAsset) external {
+        liquidityLocker.setSecondaryLiquidityAsset(_liquidityAsset);
+    }
+
+    function deleteSecondaryLiquidityAsset(address _liquidityAsset) external {
+        liquidityLocker.deleteSecondaryLiquidityAsset(_liquidityAsset);
+    }
+
+    function setSecondaryLiquidityAssets(
```

Comment:
Here and everywhere else token validity checks should be implied because of security concerns: we see it as a potential threat allowing anyone to externally set  the secondary liquidity assets.

[See previous implementation](https://github.com/groksmith/helios-contracts/blob/31fc69b7373c7b5b0bf733a9049001769acd8114/contracts/pool/Pool.sol#L88)

Reply:
Like anyone can call setSecondaryLiquidityAssets